config/src/main/java/org/hyperledger/besu/config/CliqueConfigOptions.java

```diff
@@ -59,13 +66,40 @@ public int getBlockPeriodSeconds() {
         cliqueConfigRoot, "blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
 
+  public Optional<String> getBuilderApiEndpoint() {
+    String endpoint = JsonUtil.getString(cliqueConfigRoot, "builderapiendpoint", "");
+    LOG.info("endpoint: {}", endpoint);
+    if (endpoint.isEmpty()) {
+      return Optional.empty();
+    }
+    return Optional.of(endpoint);
+  }
+
+  public Optional<BLSPublicKey> getProposerPubKey() {
+    String pubKeyHex = JsonUtil.getString(cliqueConfigRoot, "proposerpubkey", "");
```

Comment:
IMO it's not worth to be concise in config
```suggestion
    String pubKeyHex = JsonUtil.getString(cliqueConfigRoot, "proposerBlsPublicKeyHex", "");
```

Reply:
same for this one